### PR TITLE
CTECH-1501: Adding A2B report to uni module 4.1

### PR DESCRIPTION
**Pull request by JamesALeedham**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

As title.

### CODE REVIEW

File: university/module4/module4-1.ipynb [hunk 1]
```diff
@@ -49,8 +49,8 @@
        "    <tr>\n",
```
Comment (rizwansaeed) [inline university/module4/module4-1.ipynb:49]: <p>I think we should rephrase this to drop the "doesn't do X" to simply "To aggregate xxx" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

[the remark this answers is not in the document]
Comment (JamesALeedham) [reply university/module4/module4-1.ipynb:49]: <p>Done.</p>

[the remark this answers is not in the document]
Comment (JamesALeedham) [reply university/module4/module4-1.ipynb:49]: <p>Done.</p>
